modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactory.java

```diff
@@ -311,11 +312,11 @@ protected SearchScopePreference getSearchScopePreference(
 				SearchBarPortletDestinationUtil.isSameDestination(
 					headerPortletPreferencesOptional.get(), themeDisplay)) {
 
-				Optional<String> optional = searchSettings.getScope();
+				String scope = searchSettings.getScope();
```

Comment:
@julschong Please make additional change to flip conditions in the methods you modified in this class, making these methods contain only 1-level if-statement. You can use IntelliJ IDEA feature "Flip If condition" (Alt+Enter when your cursor is on `if`).